### PR TITLE
920 Migrate AccessModalContent to package UI

### LINKED ISSUE #920: [FE] Migrate AccessModalContent to package UI

### PR DESCRIPTION
**Pull request by octapf**
Close #920

### CODE REVIEW

**Review by REALcuba — DISMISSED** [review]

File: packages/ui/src/__tests__/molecules/AccessModalContent.test.tsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
+import { vi } from 'vitest'
+import { fireEvent, render, screen, waitFor } from '../test-utils'
```
Comment (NataliaR-BCN) [inline packages/ui/src/__tests__/molecules/AccessModalContent.test.tsx:2]: Or import directly from '@testing-library/react'

File: packages/ui/src/stories/molecules/AccessModalContent.stories.ts [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import type { Meta, StoryObj } from '@storybook/react'
+import { AccessModalContent } from '../../components/molecules'
+import customImg from '../assets/lock-dynamic-color.svg'
```
Comment (kevinmamaqi) [inline packages/ui/src/stories/molecules/AccessModalContent.stories.ts:3]: Pass as props

File: packages/ui/src/components/molecules/AccessModalContent.tsx [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+`
+type TAccessModal = {
+  handleAccessModal: () => void
+  handleLoginModal: () => void
+  handleRegisterModal: () => void
+  title: string
+  userMsg: string
+  registerBtnTitle: string
+  loginBtnTitle: string
+  svgSrc: string
```
Comment (kevinmamaqi) [inline packages/ui/src/components/molecules/AccessModalContent.tsx:37]: img

type TImg {
src: string
alt: string
}

File: packages/ui/src/stories/molecules/AccessModalContent.stories.ts [hunk 4]
```diff
@@ -0,0 +1,57 @@
+import type { Meta, StoryObj } from '@storybook/react'
```
Comment (kevinmamaqi) [inline packages/ui/src/stories/molecules/AccessModalContent.stories.ts:1]: Crear carpeta `AccessModalContent/index.tsx`

**Review by kevinmamaqi — CHANGES_REQUESTED** [review]

**Review by kevinmamaqi — DISMISSED** [review]

**Review by NataliaR-BCN — APPROVED** [review]

**Review by NataliaR-BCN — APPROVED** [review]

**Review by marmorrei — APPROVED** [review]

**Pull request merged**